src/index.js

```diff
@@ -43,7 +43,9 @@ baseHandler.post = function(params, callback) {
 
   var inputDoc = JSON.parse(fs.readFileSync(__dirname + '/json/state_machine_input.json', {encoding:'utf8'}));
   var ruleJson = JSON.parse(fs.readFileSync(__dirname + '/json/default_config_rules.json', {encoding:'utf8'}));
-  
+
+  params.platform = "ukgc"
```

Comment:
Will this platform value be the hardcoded value or will be passed to the proxy lambda function from the Sails API framework ?

Reply:
This was added for testing and has been removed.